tests/health-check.test.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+const chai = require('chai'),
+      chaiExec = require("@jsdevtools/chai-exec"), // Allows us to assert CLI commands
+      assert = chai.assert,
+      expect = chai.expect;
+const fs = require('fs'), // Node file system
+      { exec, execSync } = require("child_process"); // Node exec function that allows us to run other commands
+
+// Register @jsdevtools/chai-exec
+chai.use(chaiExec);
+
+/* Returns true or false depending if a directory exists or not */
+function checkDirectoryExists(directory) {
+  try {
+    if (fs.existsSync(directory)) {
+      return true
+    }
+  } catch(err) {
+    return false
+  }
+}
+
+/* Alias for checkDirectoryExists() so that code is more readable */
+function checkFileExists(file) {
+  return checkDirectoryExists(file)
+}
+
+/* Returns true if a file contains string */
+function fileContains(file, text) {
+  try {
+    const data = fs.readFileSync(file, {encoding: 'utf8'})
+    if(data.includes(text)) {
+      return true
+    } else {
+      return false
+    }
```

Comment:
I have a feeling you can replace this whole bit with a one liner: return fs.readFileSync(file, {encoding: 'utf8'}).includes(text)